src/main/java/com/sngular/kloadgen/sampler/schemaregistry/adapter/impl/ApicurioParsedSchemaMetadata.java

```diff
@@ -0,0 +1,30 @@
+package com.sngular.kloadgen.sampler.schemaregistry.adapter.impl;
+
+import com.sngular.kloadgen.sampler.schemaregistry.schema.ApicurioParsedSchema;
+import lombok.Getter;
+
+@Getter
+public class ApicurioParsedSchemaMetadata extends ParsedSchemaAdapter {
+
+  private final String type;
+
+  private final Object schema;
+
+  private final String rawSchema;
+
+  public ApicurioParsedSchemaMetadata(final ApicurioParsedSchema apicurioParsedSchema) {
```

Comment:
Constructor argument must be an io.apicurio.registry.resolver.ParsedSchema object type

Reply:
done